pkg/mapper/configmap/mapper.go

```diff
@@ -13,7 +13,10 @@ type ConfigMapMapper struct {
 
 var _ mapper.Mapper = &ConfigMapMapper{}
 
+var LenientYamlEKS = false
```

Comment:
What's the reasoning for this global exported variable here?

Reply:
it is only meaningful in this file. it is a short term fix for EKS configmap

Reply:
Thoughts on making it unexportable then if it's only meaningful in this package?